### ISSUE TITLE
Build failures

### ISSUE THREAD

**Issue by jpfeuffer** [OpenMS/OpenMS#6179]
```
[src/openms/source/ANALYSIS/OPENSWATH/TargetedSpectraExtractor.cpp:292](https://github.com/OpenMS/OpenMS/blob/master/src/openms/source/ANALYSIS/OPENSWATH/TargetedSpectraExtractor.cpp#L292):9: error: unused variable ‘unknown_feature_counter’ [-Werror=unused-variable]
```

@bertrandboudaud

**Comment by bertrandboudaud** [comment]
Sorry for that, I'm not sure how to check that kind of error in your CI.

PR is here https://github.com/OpenMS/OpenMS/pull/6180

**Comment by jpfeuffer** [comment]
No problem, we should make warn=error on PR CI for gcc as well.
For now you have to check for warnings as well.

Other compilers just have too many warnings anyway.

**Comment by enetz** [comment]
fixed by #6180
